app/models/efile_error.rb

```diff
@@ -13,7 +13,8 @@
 #  updated_at :datetime         not null
 #
 class EfileError < ApplicationRecord
-  def self.pdf_generation_error
-    create_or_find_by!(source: :internal, code: 'PDF-1040', message: 'Failed to generate PDF Form 1040.')
-  end
+  has_rich_text :description_en
```

Comment:
The examples in the screenshot all are plain text. This is rich text. Would it suffice to use plain text here? If so, I encourage that.

ActionText is new and it scares me! I am willing to learn not to be scared, and this could be irrational on my part, but that's my feeling at this moment. There are a lot of moving pieces of our app, and my general anxiety level is higher now than average.

I see at https://github.com/basecamp/trix/releases that the most recent release involved a HTML sanitization fixes, which on average, means the next one will as well, and therefore it'll be important to update it frequently.

I also see there's support for images in here. What's the significance/purpose of that?

Reply:
the examples in the screenshots are not indicative of the text that they want to put in -- they're multi-paragraph and have links to other places. It essentially needs to either be WSYIWIG or markdown, and any time i've added an admin feature for non-technical folks that uses markdown, they have hated me forever.

While ActionText new, Rails is _generally_ thoughtful and careful about promoting things into Rails, so i feel confident that it will be okay. Also, the implementation is limited enough that if we end up hating it, we can rip it out fairly easily and store the contents of these messages elsehow. 

The other option would be to grab another WYSIWYG editor and save that data into a field on the model. That's what I was expecting we'd have to do until I read about ActionText and took a shot at configuring it. The way it does storage is likely a little overkill for our use case, but it does get us up and running SO much faster than if we had to roll our own solution.